quizMySQL/GameActivity.cs

```diff
@@ -0,0 +1,252 @@
+﻿using Android.App;
+using Android.Content;
+using Android.Widget;
+using Android.OS;
+using System.IO;
+using System.Data;
+using quizMySQL;
+using MySql.Data.MySqlClient;
+using System;
+using System.Collections.Generic;
+using System;
+using quizMySQL.DataBase;
+using Newtonsoft.Json;
+
+
+namespace quizMySQL
+{
+    [Activity(Label = "GameActivity", MainLauncher = false, Icon = "@drawable/icon")]
+    public class GameActivity : Activity
+    {
+        QuizGame game;
+        TextView questionNoTxt;
+        TextView question;
+        Button buttonA;
+        Button buttonB;
+        Button buttonC;
+        Button buttonD;
+
+
+        protected override void OnCreate(Bundle bundle)
+        {
+            base.OnCreate(bundle);
+            // Set our view from the "main" layout resource
+            SetContentView(Resource.Layout.Game);
+            game = new QuizGame();
+            game.database = new DbManager("Server=10.0.2.2;Database=quiz;Uid=root;Allow User Variables=True;");
+            game.populatingQuestionList(game.database);
+            assignViewToVar();
+            refreshQuestionPage();
+            buttonA.Click += delegate { btn1Click(); };
+            buttonB.Click += delegate { btn2Click(); };
+            buttonC.Click += delegate { btn3Click(); };
+            buttonD.Click += delegate { btn4Click(); };
+        }
+        public void refreshQuestionPage()
+        {
+            assignValToVar(game.currentQuestion);
+        }
+        private void assignViewToVar()
+        {
+            questionNoTxt = FindViewById<TextView>(Resource.Id.textView2);
+            question = FindViewById<TextView>(Resource.Id.textView1);
+            buttonA = FindViewById<Button>(Resource.Id.button1);
+            buttonB = FindViewById<Button>(Resource.Id.button2);
+            buttonC = FindViewById<Button>(Resource.Id.button3);
+            buttonD = FindViewById<Button>(Resource.Id.button4);
+        }
+        private void assignValToVar(int questionNumber)
+        {
+            questionNoTxt.Text = game.updateCurrentQuestionTxt();
+            question.Text = game.getQuestion(questionNumber).question;
+            buttonA.Text = game.getQuestion(questionNumber).answerA;
+            buttonB.Text = game.getQuestion(questionNumber).answerB;
+            buttonC.Text = game.getQuestion(questionNumber).answerC;
+            buttonD.Text = game.getQuestion(questionNumber).answerD;
+
+        }
+#region btnsClick
+        private void btn1Click()
+        {   //Checking if A is correct
+            if(game.checkClick("A"))
```

Comment:
a lot of copy&paste code, should be refactored into a single method called from within the btn click handlers